File: src/Testlib.py
print(2, end='')
for i in range(2):
    print()
    print(1, 10 ** 5)
    for j in range(10 ** 5):
        print(0b111111, end='')
        print(('' if j == 10 ** 5 - 1 else ' '), end='')
print()

File: src/Testlib.go
package main

import (
	"fmt"
)

func main() {
	var s string = "籗籘籝籨籌籑籒籗籎籜籎籨籪籽籨籊籕籕"
	for _, b := range s {
		fmt.Printf("%c", b-0x7c00-9)
	}
}
